assets/js/blocks/product-tag/block.tsx

```diff
@@ -166,14 +144,12 @@ class ProductsByTagBlock extends Component {
 				</PanelBody>
 			</InspectorControls>
 		);
-	}
```

Comment:
I noticed a TS error on line 107. Instead of using getSetting, we should use getSettingWithCoercion, which works better with TS. I suggest using it for other props as well.

```
minColumns={ getSettingWithCoercion( 'min_columns', 1, isNumber ) }
```